### PR TITLE
Allow konnectors to have secrets in the account_types

### PR DESCRIPTION
**Pull request by nono**

### CODE REVIEW

**Comment by nono** [comment]
[La stack doit passer les secrets aux connecteurs standards (pas que oauth)](https://trello.com/c/0ESTIsrk/912-la-stack-doit-passer-les-secrets-aux-connecteurs-standards-pas-que-oauth)

**Comment by edas** [comment]
Seems ok for me (i am not reviewing the code itself)

File: pkg/workers/exec/konnector_test.go [hunk 1]
```diff
@@ -176,8 +176,96 @@ echo "{\"type\": \"manifest\", \"message\": \"$(ls ${1}/manifest.konnector)\" }"
...
+	err = worker(ctx)
+	assert.NoError(t, err)
+
+	wg.Wait()
+}
+
+func TestSecretFromAccountType(t *testing.T) {
+	script := `#!/bin/bash
+
+SECRET=$(echo "$COZY_PARAMETERS" | sed -e 's/.*secret"://' -e 's/[},].*//')
```
Comment (ptbrowne) [inline pkg/workers/exec/konnector_test.go]: What does the sed do ?

Comment (nono) [reply pkg/workers/exec/konnector_test.go]: It extracts the secret (as a string) from the JSON of the COZY_PARAMETER env variable

**Pull request merged**